hoot-core-test/src/test/cpp/hoot/core/TestUtils.h

```diff
@@ -117,17 +117,19 @@ class TestUtils
   static bool compareMaps(const QString& map1, const QString map2);
 
   static NodePtr createNode(OsmMapPtr map, Status status, double x, double y,
-    Meters circularError = ElementData::CIRCULAR_ERROR_DEFAULT, Tags tags = Tags());
+    Meters circularError = ConfigOptions().getCircularErrorDefaultValue(), Tags tags = Tags());
 
```

Comment:
There are a ton of changes similar to this.  The reason for the changes is because that the way circular.error.default.value was being set wouldn't allow it to be passed in from the command line as an option. Doing this is useful when you want to see the effect circular error calculations have on convert and conflate commands...both output quality and runtime performance.